### DISCUSSION TITLE
The request be rejected

### DISCUSSION THREAD

**Discussion by lqx3089** [community/community#41048] [GitHub Education] [answered]
My request to get the Student Developer Pack was be rejected and the rejected message was
![image](https://user-images.githubusercontent.com/76772873/206321788-74f62647-7ced-4cf2-8342-df063adc8631.png)

---

However, I have checked the picture I uploaded, and I did not find any problems raised by the rejection message. However, in order to ensure my privacy, I marked my ID number and student ID number on the picture with a Mosaic. I want to know whether it is because of this that I was rejected

**Comment by lqx3089** [comment]
and here are my picture
![lqx（公章）](https://user-images.githubusercontent.com/76772873/206323748-aadcc546-d2cd-4927-b9ca-f3902578050e.jpeg)

**Comment by samus-aran** [comment, marked as answer]
Hi @lqx3089  👋 , we're happy you're here! 

Thank you for providing your rejection reasons and additional context to your application. Let's see how we can assist.

1. The main rejection reason appears to be that you've submitted insufficient evidence. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Dated Student ID being at the top. If you don't have a Student ID do try and acquire evidence that is positioned higher rather than lower in that option list. 
2. Your evidence appears to be missing a date to be deemed as "dated" evidence. Perhaps you can request your school to re-issue it with a date.
3. Ensure you have updated your GitHub profile information that corresponds to your application evidence. The verification process does take into account your GitHub username, email, name and other aspects.

Let us know how you get on 👍

**Reply by lqx3089** [reply]
thank you for your answer, The certificate provided by my school made a very humble mistake and lacked the date of proof. Now I ask my school to provide me with a new document with the exact date. Thanks again for your answer
